cmd/flightctl/main.go

```diff
@@ -34,6 +34,7 @@ func NewFlightCtlCommand() *cobra.Command {
 	cmd.AddCommand(cli.NewConsoleCmd())
 	cmd.AddCommand(cli.NewCmdCompletion())
 	cmd.AddCommand(cli.NewCmdEnrollmentConfig())
+	cmd.AddCommand(cli.NewCmdCertificate())
```

Comment:
I suppose it depends on whether we want to have the `apply -f` command (which mimics kubernetes) available as an alternative to this. If we do want it available, the enrollmentconfig command is the only way to produce the config file that is needed in that flow.

```
# Example 1
flightctl apply -f examples/csr.yaml
flightctl approve csr/mycsr # <-- this will go away with auto-approve
enrollmentconfig mycsr -p myeckey.pem

# Example 2 - equivalent to above
flightctl certificate request 
```

Note that if for some reason the user wants to manually approve an ecert (will we ever want to support that?) or hits `ctrl+C` during the `certificate request` before the CSR is approved and the files are written out, there isn't a good way to get the enrollment config asynchronously without the enrollmentconfig API.

Reply:
And, actually, the enrollmentconfig API requires specifying a key - so if we do want to make it an option to get the info with `enrollmentconfig` after a `certificate request`, then the key should be written out before the CSR is sent so that it is available even if the command doesn't complete. We should also probably provide an option to encrypt the keys.

Reply:
Based on today's sync we agreed there should be some API to fetch this type of data, but the `enrollmentconfig` API may morph into something more generic such as a discovery API. In any case changes to `enrollmentconfig` can be in a later PR.